README.md

```diff
@@ -103,6 +103,8 @@ type MswParameter = {
 }
 ```
 
+_**Note:** Handler inheritance only works when the root `handlers` is an object; arrays of handlers will not be merged._
```

Comment:
@yannbf, this is something we need to reconsider. What is the argument against merging array handlers? I'd expect Storybook to use `.unshift()` and be done with this.